pkgs/development/libraries/hunspell/dictionaries.nix

```diff
@@ -821,4 +823,27 @@ in rec {
     readmeFile = "README_he_IL.txt";
     license = with lib.licenses; [ agpl3Plus ];
   };
+
+  /* THAI */
+
+  th_TH = th-th;
+  th-th = mkDict {
+    pname = "hunspell-dict-th-th";
```

Comment:
I think you can define just `shortName` as `"th-th"` and omit `pname`

Reply:
I think `mkDict` doesn't take the `shortName`, no?

Reply:
ah right. it might be specific `mkDictFromWordlist`